presto-main/src/main/java/io/prestosql/execution/QueryManagerConfig.java

```diff
@@ -228,6 +231,19 @@ public QueryManagerConfig setQueryManagerExecutorPoolSize(int queryManagerExecut
         return this;
     }
 
+    @NotNull
+    public Optional<DataSize> getQueryMaxDataSize()
+    {
+        return queryMaxDataSize;
+    }
+
+    @Config("query.max-data-size")
+    public QueryManagerConfig setQueryMaxDataSize(Optional<DataSize> queryMaxDataSize)
```

Comment:
The argument should be `DataSize`. Otherwise, Presto server will throw the following exception during the startup. 
```
1) Error: Could not coerce value '10TB' to java.util.Optional (property 'query.max-data-size') in order to call [public io.prestosql.execution.QueryManagerConfig io.prestosql.execution.QueryManagerConfig.setQueryMaxDataSize(java.util.Optional<io.airlift.units.DataSize>)]

2) Configuration property 'query.max-data-size' was not used
  at io.airlift.bootstrap.Bootstrap.lambda$initialize$2(Bootstrap.java:238)
```